src/test/java/dev/snowdrop/release/services/TestProfiles.java

```diff
@@ -0,0 +1,34 @@
+package dev.snowdrop.release.services;
+
+import io.quarkus.test.junit.QuarkusTestProfile;
+
+import java.util.Collections;
+import java.util.Set;
+
+/**
+ * @author <a href="antcosta@redhat.com">Antonio Costa</a>
+ */
+public class TestProfiles  {
+    public static class NoTags implements QuarkusTestProfile {
```

Comment:
Can we remove `NoTags` profile? it is never used, isn't it?

Reply:
done